src/ros_can_nodes/scripts/generate_message_properties_map.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+import sys
+import os.path
+import glob
+import itertools
+
+import roslib.msgs
+import roslib.packages
+from rospkg import RosPack
+
+try:
+    from cStringIO import StringIO #Python 2.x
+except ImportError:
+    from io import StringIO #Python 3.x
+
+
+hpp_contents =\
```

Comment:
not sure if i should generate the hpp, given that its contents are static anyway

Reply:
I'd leave it out of the repo (just have this script) - it won't be static